_pages/contact.md

```diff
@@ -1,6 +0,0 @@
----
-title: Contact
```

Comment:
## Delete: contact page
Decided to delete this and instead list our contact info on either the home/about page or the team page.